utils/gyb_syntax_support/Traits.py

```diff
@@ -0,0 +1,24 @@
+from Child import Child
+
+
+class Trait(object):
+    def __init__(self, trait_name, children):
+        self.trait_name = trait_name
+        self.children = children
+
+
+TRAITS = [
+    Trait('DeclGroup',
```

Comment:
I think traits should also be postfixed with "`Syntax`".

Could you make this `DeclGroupSyntax`?
Or make them `DeclGroup` and `Braced`, and postfix in the client?
```swift
public protocol ${trait.trait_name}Syntax {
```

Reply:
Sure, I will rename this to `DeclGroupSyntax`.